routes/index.js

```diff
@@ -54,14 +54,15 @@ var routes = {
         amount: transaction.amount,
         stripeToken: transaction.stripeToken,
         email: transaction.email,
+        currency: transaction.currency,
         metadata: {
-          firstname: transaction.firstName,
-          lastname: transaction.lastName,
+          firstname: transaction.first,
```

Comment:
I almost wonder if we shouldn't just be creating a Stripe customer object first...